stdlib/Profile/src/Profile.jl

```diff
@@ -1139,4 +1139,6 @@ function warning_empty(;summary = false)
     end
 end
 
+include("Allocs.jl")
+
```

Comment:
Does this need to go at the end? I think usually the standard is to put `include`s at the top of the module. If it needs to use names from the parent module, it'd probably be preferable to move the whole of the Profile.jl file into another file (call it `cpu_profile.jl`?) and include that before including this?

But ideally you can just put this at the top (after the imports/usings) and it'll just work? 🤞